debian_config.json

```diff
@@ -35,12 +35,13 @@
         "bin/dotnet" : "usr/bin/dotnet",
         "bin/dotnet-compile" : "usr/bin/dotnet-compile",
         "bin/dotnet-compile-csc" : "usr/bin/dotnet-compile-csc",
+        "bin/dotnet-compile-native" : "/usr/bin/dotnet-compile-native",
```

Comment:
This is the wrong file, and I need to delete it (going to do now). Check packaging/debian/debian_config.json